app.js

```diff
@@ -31,11 +41,13 @@ app.use(express.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 // Public routes which don't require a TOKEN
-app.post("/api/v1/auth", getAuthToken)
+app.post("/api/v1/auth/api", getAuthToken)
+app.get("/api/v1/auth/google", googleAuthRedirect);
+app.get("/api/v1/auth/google/callback", googleAuthCallback);
 
 // Routes required to have a valid TOKEN
-// app.use("/api/v1", checkJwt, router);
-app.use("/api/v1", router); //TODO uncomment above line and remove this line
+app.use("/api/v1", checkGoogleAccessToken, router); //TODO For development, comment this line
+// app.use("/api/v1", router); //TODO For development, uncomment this line
```

Comment:
we could create an env variable, to turn on and off the auth check, so we don't have to comment/uncomment this

Reply:
DEV_MODE created on env variables